.github/workflows/tests.yml

```diff
@@ -21,8 +21,79 @@ on:
       - 'doc/**'
 
 jobs:
+  no-meta:
+    strategy:
+       matrix:
+          OS: [macos-10.15, ubuntu-20.04]
+          CXXCOMPILER: [g++, clang++]
+    runs-on: ${{ matrix.OS }}
+
+    steps:
+      - name: Homebrew 
+        if: runner.os == 'macOS'
+        run: |
+           brew update
+           brew upgrade
+           brew install gsl
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Conan
+        run: |
+          pip3 install --user conan
+          export PATH=$(python3 -c 'import site; print(site.USER_BASE + "/bin")'):$PATH
```

Comment:
I think this line can be omitted when using github actions. It was used only for travis-ci.

Reply:
I thing this has to stay. Without it I'm getting
```
/home/runner/work/_temp/24ba64e7-a44d-4bc9-a572-5d71cce1cd70.sh: line 3: conan: command not found
```